doc/changes/latest.inc

```diff
@@ -132,6 +132,8 @@ Bugs
 ~~~~
 - Fix bug with :func:`mne.io.read_raw_nihon` where latin-1 channels could not be read (:gh:`10429` by :newcontrib:`Matthias Eberlein`)
 
+- Fix bug with :func:`mne.io.read_raw_nihon so that it handles duplicates in channel names (:gh:`10431` by :newcontrib:`Matthias Eberlein`)
```

Comment:
```suggestion
- Fix bug with :func:`mne.io.read_raw_nihon` so that it handles duplicates in channel names (:gh:`10431` by :newcontrib:`Matthias Eberlein`)
```